### ISSUE TITLE
sNotz

### ISSUE THREAD

**Issue by ProblematicMikey** [sunilpaulmathew/sNotz#34]
Was there ever an option to back up notes via clouds...I'm sorry for disturbing you and at the same time I appreciate you and your most seamless utility.....Ty

**Comment by sunilpaulmathew** [comment]
Nope, as already mentioned in the app description (_sNotz is an open-source, privacy-friendly, secure, and **completely offline** application..._). Thanks
